src/duration/extractMinMaxInUnits.js

```diff
@@ -1,5 +1,5 @@
 import durationUnits from './durationUnits';
-import splitSecsToUnits from './splitSecsToUnits';
+import splitSecsToUnits from '../date/splitSecsToUnits';
```

Comment:
Then we should fix it now, else we will be merging failed PR.
How about we let the compiled code follow the repo structure?

This will break other repo, but webpack will catch them